internal/rbd/rbd_util.go

```diff
@@ -1922,27 +1925,54 @@ func genVolFromVolIDWithMigration(
 	return rv, err
 }
 
+// getCOName reads the `CONAME` from the environment and returns
+// its value. In case the CONAME can not be detected, an error is returned.
+func getCOName() (string, error) {
+	co := os.Getenv(COName)
+	if co == "" {
+		return "", fmt.Errorf("%q is not set in the environment", COName)
```

Comment:
Not everyone will set this by default. you don't need to return an error message from there. you can read it once when starting the driver and just log it and keep it in memory and use it.